composer.json

```diff
@@ -6,7 +6,7 @@
     "license": "MIT",
     "require": {
         "php": "^7.2",
-        "sylius/sylius": "^1.4"
+        "sylius/sylius": "^1.9"
```

Comment:
It seems to work on sylius 1.6, is it really necessary to leave this requirement?

Reply:
It's because I am on php7.4 with Sylius 1.9 so I cannot test it below but if it's ok, no problem ^^

Reply:
Perfect, can you leave 1.4 compatibility? After that I merge, thank you!